sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Internals/Profiling/ProfilingSessionEventSource.cs

```diff
@@ -37,10 +53,45 @@ internal class ProfilingSessionEventSource : EventSource
         /// </summary>
         public static ProfilingSessionEventSource Current { get; } = new();
 
+        /// <summary>
+        /// Keywords for this EventSource.
+        /// </summary>
+        public static class Keywords
+        {
+            /// <summary>
+            /// Emits the OpenTelemetry Resource attributes. These attributes
+            /// are emitted exactly once at Informational level.
+            /// </summary>
+            public const EventKeywords ResourceAttributes = (EventKeywords)0b_1;
+
+            /// <summary>
+            /// Emits start/stop events for activities.
+            /// </summary>
+            public const EventKeywords Activities = (EventKeywords)0b_10;
```

Comment:
While researching some other issues, I notice that the OpenTelemetry base SDK has its own OpenTelemetry-Sdk EventSource with ActivityStarted/Stopped.

I'm thinking, therefore, that I don't need to repeat it here. The profiler can enable that EventSource (ETW provider) instead.